PTBR/Recorded_Media_PTBR.txt

```diff
@@ -2341,144 +2341,144 @@ RM_64fa976d-ad89-4845-b0a3-3068ca1b1867 = "And season to taste!"
+RM_26d6fbe3-6fa0-4add-b3cd-9f0b4dc05739 = "Cinco tábuas! Quatro pregos! Isso aí!"
+RM_39a0111d-0529-4fde-86fc-722a201173a1 = "Senhoras! Vocês podem a-cre-di-tar nisso?"
+RM_90317c24-3225-46b6-8df2-a0bc018e6ede = "Agora que tal umas prateleiras?"
+RM_7bb34ff8-504e-46d3-b51a-efdcdc9d8038 = "Um lugar para suas joias, suas preciosas coisas?"
+RM_8d3e42db-0a5f-4aa1-ad36-bcec0df8a0dd = "DUAS TÁBUAS! QUATRO PREGOS! VAI!"
+RM_bb0ae3ec-4b14-445b-83d0-a17175e57236 = "Só preciso..."
 RM_536d2a9b-2b2f-4683-b517-c21da4c95552 = "*bang*"
-RM_fc3e6ea2-e775-4bec-ab62-649bc06ba729 = "That's it."
-RM_b8960705-6bb2-424f-8b6d-ea7d22a83047 = "Oh hey, sick of folk seeing you sunbathin'?"
-RM_ba949727-2b3a-46ba-a75b-74af46c0b59a = "I was. Get it all the time."
-RM_f7948221-372c-4c26-b14e-bf0867cc1316 = "So I built me a fence."
-RM_bd80fa61-7f4d-4a57-948f-76db019dea6f = "I was all: two planks, three nails, bang, bang."
-RM_1ad72c61-855d-4374-aac5-482849b70c31 = "And here she is!"
-RM_4ad36743-ad25-4116-b35e-df46a927ae10 = "So if you'll excuse me, I'll just..."
-RM_5f7b7bf7-7439-40ac-a3f4-1ac739931045 = "Get comfy with my shirt off..."
-RM_d9ea7d9c-4a56-46ce-bddd-c35f8b87d2cf = "Next time on Woodcraft? Doors!"
-RM_cad9e8aa-1f6c-4e47-a588-430fb43287df = "Can I hear a yee-hah?"
-RM_76f3676c-493f-478e-b228-1dc7c1f0e0af = "VHS: Woodcraft E5"
-RM_bc33e81d-30f1-43a3-b4f0-bb039a3a1207 = "Episode 5: Doors"
-RM_31f0e287-8144-4d80-b1c2-5fce7c7245e8 = "Well hey! Y'all just caught me working out!"
-RM_ba8abeaf-af63-4fe0-8a2b-72610c608f46 = "Got a real sweat going on here!"
-RM_a55409b8-a393-4589-b95d-4c7debc93d65 = "Next time I'll shut the door..."
-RM_17e44616-3875-4909-8bd1-366bf7f30f05 = "But hey! There's no door!"
-RM_d0e63cf9-f6fc-4eff-8f66-e0f801105aad = "Guess this happened to just the right guy!"
-RM_54fe89cd-2404-48b5-af94-30d2deb23c01 = "You know the drill!"
-RM_b6cf63d6-2a8e-4c1a-9b15-995ac7d51fdb = "Four planks! Four nails! One knob! One hinge!"
+RM_fc3e6ea2-e775-4bec-ab62-649bc06ba729 = "É isso."
+RM_b8960705-6bb2-424f-8b6d-ea7d22a83047 = "Oh! Ei, está cansado de pessoas vendo você tomar sol'?"
+RM_ba949727-2b3a-46ba-a75b-74af46c0b59a = "Eu estava. Era o tempo inteiro."
+RM_f7948221-372c-4c26-b14e-bf0867cc1316 = "Então eu construí uma cerca."
+RM_bd80fa61-7f4d-4a57-948f-76db019dea6f = "Isso é tudo o que precisa: quatro tábuas, três pregos, bang, bang."
+RM_1ad72c61-855d-4374-aac5-482849b70c31 = "Aqui está ela!"
+RM_4ad36743-ad25-4116-b35e-df46a927ae10 = "Então, se me der licença, Eu vou apenas..."
+RM_5f7b7bf7-7439-40ac-a3f4-1ac739931045 = "Ficar confortável sem minha camisa..."
+RM_d9ea7d9c-4a56-46ce-bddd-c35f8b87d2cf = "No próximo programa de Artesanato em Madeira? Portas!"
+RM_cad9e8aa-1f6c-4e47-a588-430fb43287df = "Posso ouvir um iupii?"
+RM_76f3676c-493f-478e-b228-1dc7c1f0e0af = "VHS: Artesanato em Madeira EP5"
+RM_bc33e81d-30f1-43a3-b4f0-bb039a3a1207 = "Episódio 5: Porta"
+RM_31f0e287-8144-4d80-b1c2-5fce7c7245e8 = "Bem olá! Vocês acabaram de me pegar malhando!"
```

Comment:
"**Bem, olá**! Vocês acabaram de me pegar malhando!"

A vírgula é importante aqui. Imaginamos que ele está malhando e subitamente inicia o programa. É como se expressasse de maneira surpresa no início.